recipes/mirotalk-c2c/index.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+module.exports = Ferdium => class MiroTalkC2C extends Ferdium {
+	overrideUserAgent() {
+		return window.navigator.userAgent.replace(/(Ferdium|Electron)(\S+\s)/g, '').trim();
```

Comment:
Is this really required? If so, please specify why

Reply:
Here not sorry, I'm going to fix.